Index: src/main/java/com/sylleryum/meajudaaajudar/security/LibraryApiSecurityConfig.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/com/sylleryum/meajudaaajudar/security/LibraryApiSecurityConfig.java	(date 1601408350490)
+++ src/main/java/com/sylleryum/meajudaaajudar/security/LibraryApiSecurityConfig.java	(date 1601408350490)
@@ -24,7 +24,7 @@
             httpSecurity.cors().and().csrf().disable().authorizeRequests()
                     .antMatchers(HttpMethod.POST, SecurityConstants.NEW_USER_REGISTRATION_URL).permitAll()
                     .antMatchers(HttpMethod.GET, "/v1/users/search").permitAll()
-                    .antMatchers(HttpMethod.GET, "/v1/**").hasRole()
+                    .antMatchers(HttpMethod.GET, "/v1/estados/**").hasRole("admin")
                     .anyRequest().authenticated()
                     .and()
                     .addFilter(new JwtAuthenticationFilter(authenticationManager()))
